src/ebay-listbox-button/listbox-button.tsx

```diff
@@ -202,6 +214,11 @@ const ListboxButton: FC<EbayListboxButtonProps> = ({
     })
     const expandBtnTextId = prefixId && 'expand-btn-text'
 
+    useEffect(() => {
+        if (expanded) onExpand()
+        else onCollapse()
+    }, [expanded])
+
```

Comment:
This will also run on first render. Can we instead do it on the events?

Reply:
fixed (improved condition instead, otherwise it would need too many changes)